infor-design/enterprise
# 3335 - Fix inconsistent line spacing with editor [v4.25.x]

**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Fixed inconsistent line spacing with Editor.

**Related github/jira issue (required)**:
Closes #3335

**Steps necessary to review your pull request (required)**:
- Pull this branch and build/run the demo app
- Open chrome browser
- Navigate to http://localhost:4000/components/editor/example-index.html
- Click in editor very first and press `Enter`
- Up arrow to take curser line up and type `Test`
- Select that text `Test` and make `Header1`
- Click next to text `Test` and press `Enter`
- Cursor should go line down, type `123`
- Click on `View Source` to switch html mode
- See in very top `Test` should be in `h3` and `123` should be in `p` tag

**Included in this Pull Request**:
